assignment08/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+    include "includes/header.php";
+
+    $first_name = 'Dissa';
+    echo $first_name;
+    echo "<br>";
+    $last_name = 'Kondo';
+    echo $last_name;
+    echo "<br>";
+    $my_age = 36;
+    echo $my_age;
+    echo "<br>";
+    $my_weight = 170;
+    echo $my_weight;
+    $val = 1/2.2;
```

Comment:
this is just going to store the value of 1 divded by 2.2. That should be the calculation (times the weight in pounds) that you do in your function.